package.json

```diff
@@ -18,53 +14,21 @@
     "crypto"
   ],
   "scripts": {
-    "prebuild": "rm -rf lib && rm -rf dist",
-    "build": "yarn run build:node && yarn run build:web && yarn run build:types",
-    "bundle": "webpack --config webpack.bundle.config.js",
-    "build:node": "babel --no-babelrc --config-file ./babel.config.json src --copy-files -x .ts -d dist --source-maps",
-    "build:web": "babel --no-babelrc --config-file ./babel.web.config.json src --copy-files -x .ts -d lib --source-maps",
-    "build:types": "tsc --declaration --outDir dist --emitDeclarationOnly",
-    "proto:gen": "pbjs -t static-module -o ./src/proto/payload.js ./src/proto/payload.proto && pbts -o ./src/proto/payload.d.ts ./src/proto/payload.js && yarn run lint --fix",
-    "check-types": "tsc --noEmit",
-    "lint": "eslint --ext .ts src/",
-    "pretest": "yarn check-types",
-    "test": "yarn run test:node && yarn run test:web",
-    "test:node": "mocha -r ./babel-register.js \"test/**/*.test.ts\"",
-    "test:web": "karma start"
+    "build": "aegir build --ts",
+    "lint": "aegir lint --ts",
+    "test": "aegir test --ts",
+    "test:node": "aegir test -t node --ts",
+    "test:browser": "aegir test -t browser --ts",
+    "proto:gen": "pbjs -t static-module -o ./src/proto/payload.js ./src/proto/payload.proto && pbts -o ./src/proto/payload.d.ts ./src/proto/payload.js && yarn run lint --fix"
   },
   "devDependencies": {
-    "@babel/cli": "^7.6.4",
-    "@babel/core": "^7.6.4",
-    "@babel/plugin-proposal-async-generator-functions": "^7.7.0",
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/plugin-proposal-export-default-from": "^7.8.3",
-    "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/preset-typescript": "^7.6.0",
-    "@babel/register": "^7.6.2",
-    "@babel/runtime": "^7.6.3",
     "@types/bl": "^2.1.0",
     "@types/chai": "^4.2.4",
     "@types/mocha": "^5.2.7",
-    "@typescript-eslint/eslint-plugin": "^2.6.0",
-    "@typescript-eslint/parser": "^2.6.0",
-    "babel-loader": "^8.1.0",
+    "aegir": "ipfs/aegir#feat/cosmiconfig",
```

Comment:
Ups, This got merged without aegir updated

Reply:
Im gonna open few pr with some nitpicks and lint fixes so I will update before next release 🙂